app/services/calculations/export.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+module Calculations
+  class Export < Callable
+    require 'csv'
+
+    def initialize(object:)
+      @object = object
+    end
+
+    def call
+      generate_csv
```

Comment:
better place content of `generate_csv` here, but instead of block pass `&method(:method_name)`